tests/backend/test_states.py

```diff
@@ -354,31 +354,32 @@ def test_number_expectation_displaced_squeezed(self, setup_backend, tol, batch_s
         n0 = np.sinh(r0) ** 2 + np.abs(a0) ** 2
         n1 = np.sinh(r1) ** 2 + np.abs(a1) ** 2
 
-        def analytic_var(a, r, phi):
+        def squared_term(a, r, phi):
             magnitude_squared = np.abs(a) ** 2
-            var_ex = - magnitude_squared + magnitude_squared ** 2 + 2 *\
+            squared_term = - magnitude_squared + magnitude_squared ** 2 + 2 *\
                 magnitude_squared*np.cosh(2*r) - np.exp(-1j*phi) * a ** 2 *\
                 np.cosh(r)*np.sinh(r) - np.exp(1j* phi) * np.conj(a) **2 *\
-                np.cosh(r)*np.sinh(r) + np.sinh(r)**4 - (magnitude_squared +\
-                        np.conj(np.sinh(r))*np.sinh(r)) ** 2 +\
+                np.cosh(r)*np.sinh(r) + np.sinh(r)**4 +\
                 np.cosh(r)*np.sinh(r)*np.sinh(2*r)
-            return var_ex
-
-        var_tol = 0.01
-        v0 = analytic_var(a0, r0, phi0)
-        v1 = analytic_var(a1, r1, phi1)
+            return squared_term
 
+        # Increasing the tolerance when using
+        # a cutoff
+        var_tol = tol if isinstance(backend, backends.BaseGaussian) else 7 * tol
```

Comment:
@josh146 was looking at possibilities for specifying the cutoff for `setup_backend` that is being used here (going from the default of 6 to 8 would mean the original tolerance value could be used). Would there be some way to specify a custom value for the test case?

Reply:
@antalszava yes this is possible, you can simply do this in the test:

```python
backend = setup_backend(3)
# increase the cutoff to 8 for accuracy
backend.reset(cutoff_dim=8, pure=pure)
```